x-pack/plugins/apm/common/document_type.ts

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export enum ApmDocumentType {
+  TransactionMetric = 'transactionMetric',
+  ServiceMetric = 'serviceMetric',
```

Comment:
APM Server is now calling these "service transaction metrics". Should we do the same? It's a bit long so I'm fine with keeping the short version but it is a little inconsistent.

Reply:
sgtm